e2e/ansible/playbooks/hyperconverged/test-scaleup-cassandra-pods/test-scaleup-cassandra-pods.yml

```diff
@@ -0,0 +1,231 @@
+
+       - name: Download YAML for cassandra statefulset
+         get_url:
+           url: "{{ cassandra_stateful_link }}"
+           dest: "{{ result_kube_home.stdout }}/{{ stateful_yaml_alias }}"
+           force: yes
+         register: result
+         until:  "'OK' in result.msg"
+         delay: 5
+         retries: 3
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+       - name: Download YAML for cassandra loadgen
+         get_url:
+           url: "{{ cassandra_loadgen_link }}"
+           dest: "{{ result_kube_home.stdout }}/{{ loadgen_yaml_alias }}"
+           force: yes
+         register: result
+         until:  "'OK' in result.msg"
+         delay: 5
+         retries: 3
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+   
+       - name: Get the number of nodes in the cluster
+         shell: source {{ profile }}; kubectl get nodes | grep 'Ready' | wc -l
+         args:
+           executable: /bin/bash
+         register: node_out
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+       - name: Fetch the node count from stdout
+         set_fact:
+            node_count: " {{ node_out.stdout}}"
+
+
+       - name: Replace the replica count in cassandra statefulset yaml
+         replace:
+           path: "{{ result_kube_home.stdout }}/{{ stateful_yaml_alias }}"
+           regexp: 'replicas: 3'
+           replace: 'replicas: {{ (node_count) |int-2 }}'
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+       
+       - name: Replace the io_duration in cassandra loadgen yaml
+         replace:
+           path: "{{ result_kube_home.stdout }}/{{ loadgen_yaml_alias }}"
+           regexp: 'duration=5'
+           replace: 'duration={{ io_minutes }}'
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+
+       - name: Deploy openebs-operator 
+         shell: source ~/.profile; kubectl apply -f "{{ openebs_operator_link }}" 
+         args:
+           executable: /bin/bash
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       
+       - name: Check whether maya-apiserver pod is deployed
+         shell: source ~/.profile; kubectl get pods --all-namespaces | grep maya-apiserver
+         args:
+           executable: /bin/bash
+         register: result
+         until: "'Running' in result.stdout"
+         delay: 120
+         retries: 5
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}" 
+
+       - name: Deploy cassandra service
+         shell: source {{ profile }}; kubectl apply -f {{ svc_yaml_alias }} 
+         args: 
+           executable: /bin/bash
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+   
+       - name: Deploy cassandra statefulset
+         shell: source {{ profile }}; kubectl apply  -f {{ stateful_yaml_alias }}
+         args:
+           executable: /bin/bash
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
```

Comment:
Can we gather the cassandra artifacts in a list and apply in a single task using with_items